benchmarks/bvh_driver/bvh_driver.cpp

```diff
@@ -225,9 +226,9 @@ void BM_knn_search(benchmark::State &state, Spec const &spec)
     Kokkos::View<int *, DeviceType> offset("offset", 0);
     Kokkos::View<int *, DeviceType> indices("indices", 0);
     auto const start = std::chrono::high_resolution_clock::now();
-    index.query(ExecutionSpace{}, queries, indices, offset,
-                ArborX::Experimental::TraversalPolicy().setPredicateSorting(
-                    spec.sort_predicates));
+    ArborX::query(index, ExecutionSpace{}, queries, indices, offset,
```

Comment:
You don't need to qualify the free function because of ADL.  (Not asking to change)